features/Context/FieldTypeFormContext.php

```diff
@@ -163,4 +164,118 @@ public function theInputFieldsShouldBeFlaggedAsRequired()
             );
         }
     }
+
+    /**
+     * @Given the field definition is set to multiple choice
+     */
+    public function theFieldDefinitionIsSetToMultipleChoice()
+    {
+        $this->contentTypeContext->updateFieldDefinition(
+            self::$fieldIdentifier,
+            new FieldDefinitionUpdateStruct(['fieldSettings' => ['isMultiple' => true]])
+        );
+    }
+
+    /**
+     * @Then it should contain a select field
+     */
+    public function itShouldContainASelectField()
+    {
+        $this->assertElementOnPage(
+            sprintf(
+                'div.ezfield-identifier-%s fieldset select',
+                self::$fieldIdentifier
+            )
+        );
+    }
+
+    /**
+     * @Given the field definition is not set to multiple choice
```

Comment:
It's a "multiple" checkbox, yes. Otherwise I'd have used "single choice". Thanks for the clarification.